neps/utils/cli.py

```diff
@@ -928,16 +1140,70 @@ def main() -> None:
 
     # Subparser for "sample-config" command
     parser_sample_config = subparsers.add_parser(
-        "sample-config", help="Sample a configuration from existing neps state."
+        "sample-config", help="Sample configurations from the existing NePS state."
     )
     parser_sample_config.add_argument(
-        "--root-directory",
+        "worker_id",
         type=str,
-        help="Optional: The path to your root_directory. If not provided, "
-        "it will be loaded from run_config.yaml.",
+        help="The worker ID for which the configuration is being sampled.",
+    )
```

Comment:
Ahh here it is, you can probably set this to some default, such as `"cli"` or something.